rascaline-c-api/include/rascaline.hpp

```diff
@@ -454,19 +454,33 @@ class CalculationOptions {
     /// Selection of properties to compute for the samples
     LabelsSelection selected_properties = LabelsSelection::all();
 
-    /// List of gradients that should be computed. The following gradients can
-    /// be available:
+    /// List of gradients that should be computed. 
+    /// If this is `an empty list no gradients are computed. 
+    /// The following gradients are available:
     ///
     /// - ``"positions"``, for gradients of the representation with respect to
     ///   the atomic positions;
-    /// - ``"cell"``, for gradients of the representation with respect to the
-    ///   cell vectors.
+    /// - ``"cell"``, for gradients of the representation with respect to
+    ///   the cell vectors. Cell gradients are compute as
     ///
-    ///   **WARNING**: these cell gradients are different from the usual virial
-    ///   used to compute the instantaneous stress tensor (the virial is
-    ///   computed as the gradients of the representations with respect to the
-    ///   strain). To recover the typical virial, you'll need to multiply the
-    ///   cell gradients with the cell matrix.
+    ///   .. math::
```

Comment:
... and here. Is it really needed to replicate docstrings several times? This seems like a maintenance nightmare.